totalRP3/Modules/Flyway/FlywayPatches.lua

```diff
@@ -342,3 +342,77 @@ TRP3_API.flyway.patches["20"] = function()
 		end
 	end
 end
+
+TRP3_API.flyway.patches["21"] = function()
+	-- First handles the following:
+	-- About changes were always marked as unread, which is fine 9/10 of the
+	-- time. However due to an incorrect check in `onAboutReceived` we also
+	-- applied unread to empty abouts. These should not marked as unread.
+	--
+	-- Then we continue on to handling:
+	-- We were sending disabled glances data to other players unintentionally.
+
+	if not TRP3_Register then
+		return;
+	end
+
+	local profileList = TRP3_Register.profiles;
+	for profileID, profile in pairs(profileList) do
+		-- Don't check default profiles
+		if not TRP3_API.profile.isDefaultProfile(profileID) and profile.characteristics and next(profile.characteristics) ~= nil then
+			-- Handle the wrong about unread situation first.
+			local aboutData = profile.about;
+
+			-- Skip profiles that have no about data, they do not have
+			-- an unread notification as-is.
+			if aboutData then
+				-- Check if anything is filled at all in any template.
+				local filledDescription = false;
+				if aboutData.TE == 1 then
+					local templateData = aboutData.T1 or {};
+					filledDescription = templateData.TX and strtrim(templateData.TX):len() > 0;
+				elseif aboutData.TE == 2 then
+					local templateData = aboutData.T2 or {};
+					local atLeastOneFrame = false;
+					for _, frameTab in pairs(templateData) do
+						if frameTab.TX and strtrim(frameTab.TX):len() > 0 then
+							atLeastOneFrame = true;
```

Comment:
```suggestion
							atLeastOneFrame = true;
							break;
```

No need to continue looking at other parts of T2 once we've found one.